src/AutoRest.CSharp/Mgmt/Output/ResourceContainer.cs

```diff
@@ -31,11 +34,34 @@ public ResourceContainer(OperationGroup operationGroup, BuildContext<MgmtOutputL
         protected override string CreateDescription(OperationGroup operationGroup, string clientPrefix)
         {
             StringBuilder summary = new StringBuilder();
+            string ParentResourceName = GetParentResourceName();
             return string.IsNullOrWhiteSpace(operationGroup.Language.Default.Description) ?
-                $"A class representing collection of {clientPrefix} and their operations over a [ParentResource]. " :
+                $"A class representing collection of {clientPrefix} and their operations over a {ParentResourceName}." :
```

Comment:
nit: you can directly call the method from here vs having a variable to store it in
```suggestion
                $"A class representing collection of {clientPrefix} and their operations over a {GetParentResourceName()}." :
```